__tests__/app.test.js

```diff
@@ -10,12 +10,32 @@ beforeEach(() => seed(data));
 afterAll(() => db.end());
 
 describe("GET /api", () => {
-  test("200: Responds with an object detailing the documentation for each endpoint", () => {
+  test("200: serves up a json representation of all the available endpoints of the api", () => {
     return request(app)
       .get("/api")
       .expect(200)
       .then(({ body: { endpoints } }) => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
-});
\ No newline at end of file
+});
+
+describe('GET /api/topics', () => {
+  test('200: serves an array of all topics', () => {
+    return request(app)
+    .get('/api/topics')
+    .expect(200)
+    .then(({body}) => {
+      const { topics } = body;
+      expect(Array.isArray(topics)).toBe(true);
+      topics.forEach((topic) => {
+        expect(topic).toEqual(
+          expect.objectContaining({
+            slug: expect.any(String),
+            description: expect.any(String),
+          })
+        );
```

Comment:
Nice this looks really good, but what we received an empty array? How could you ensure this test fails if the array received is empty?